pkg/services/sqlstore/sqlutil/sqlutil.go

```diff
@@ -28,7 +28,7 @@ func MySQLTestDB() TestDB {
 	if port == "" {
 		port = "3306"
 	}
-	conn_str := fmt.Sprintf("grafana:password@tcp(%s:%s)/grafana_tests?collation=utf8mb4_unicode_ci&sql_mode='ANSI_QUOTES'", host, port)
+	conn_str := fmt.Sprintf("grafana:password@tcp(%s:%s)/grafana_tests?collation=utf8mb4_unicode_ci&sql_mode='ANSI_QUOTES'&parseTime=true", host, port)
```

Comment:
Is this something we want to enable? If we do I would assume we do it for the none test connection as well as we would be testing the behavior with parseTime enabled.

Reply:
yes, when the feature toggle is present, we would add also parsetime in the connection string for the non test connection https://github.com/grafana/grafana/pull/53434/files#diff-8d831fba1ec109fdc2ba8bcd44638e3d4def1b8fb725f874c2e043965ebb179aR302